Jint/Extensions/JavascriptExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System.Text;
+
+namespace Jint.Extensions
+{
+    public static class JavascriptExtensions
+    {
+        public static string UpperToLowerCamelCase(this string str)
```

Comment:
is this actually something like `ToCamelCase`?

Reply:
Yes. I explicitly named it like that to not cause confusion on when to call it.
I can rename it to `PascalCaseToCamelCase` or simply `ToCamelCase` if wanted.